nemo/collections/tts/models/magpietts.py

```diff
@@ -172,7 +172,10 @@ class SpecialAudioToken(Enum):
 
         self.decoder = transformer_2501.Transformer(**dict(cfg.decoder))
         self.final_proj = nn.Linear(cfg.decoder.d_model, self.num_audio_codebooks * self.num_all_tokens_per_codebook)
-        if cfg.get('use_local_transformer', False):
+
+        self.local_transformer_type = self.LocalTransformerType(cfg.get('local_transformer_type', 'none').lower())
+        print(f"Local transformer type: {self.local_transformer_type}")
```

Comment:
Can you switch to logging or remove this statement?
`logging.info(f"Local transformer type: {self.local_transformer_type}")` or `logging.debug(f"Local transformer type: {self.local_transformer_type}")`